### PR TITLE
IsHex and StripHex

### PR DESCRIPTION
**Pull request by rigelrozanski**

### CODE REVIEW

File: common/string.go [hunk 1]
```diff
@@ -22,3 +23,22 @@ func LeftPadString(s string, totalLength int) string {
...
+		_, err := hex.DecodeString(s[2:])
+		if err != nil {
+			return false
+		}
+		return true
+	}
+	return false
+}
+
+func StripHex(s string) string {
```
Comment (melekes) [inline common/string.go]: missing doc

File: common/string.go [hunk 2]
```diff
@@ -22,3 +23,22 @@ func LeftPadString(s string, totalLength int) string {
 	}
 	return s
 }
+
+// Returns true for non-empty hex-string prefixed with "0x"
```
Comment (melekes) [inline common/string.go]: golint would have told you doc should be in a form of "IsHex ..."

Comment (rigelrozanski) [inline common/string.go]: Thanks just installed golint addressed in PR#23

File: common/string.go (hunk 1 above)
Comment (rigelrozanski) [inline common/string.go]: Thanks just installed golint addressed in PR#23

**Pull request closed without merging**